test/container/entrypoint.sh

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 set -e
 
-if test "$(id -u)" == "0" -a "${USER_ID}" != ""; then
-  useradd -u ${USER_ID} -d /home/user -s /bin/bash ${USER_NAME:-default}
-  chown -R "${USER_NAME:-default}" ${GOCACHE}
-fi
-
```

Comment:
I’m not sure to understand this: the user is already created in the Dockerfile and Docker is explicitly run with a user ID, so this condition evaluates to false. In addition, it breaks with Podman, because Podman, while running unprivileged, appears to run as root from the container point of view (root being in fact mapped to the host user ID).